salsa_dancing_molecules/self_diffusion_coefficient.py

```diff
@@ -13,19 +13,21 @@ def get_self_diffusion_coefficient(traj_file):
 
     returns:
         self_diffusion_coefficient - float
+
     """
-    MSD = get_msd(traj_file, reference="initial")
-    self_diffusion_coefficient = calculate_self_diffusion_coefficient(MSD)
+    MSD, MSD_avr = get_msd(traj_file, reference="initial")
+    self_diffusion_coefficient = calculate_self_diffusion_coefficient(MSD, t0)
 
     return self_diffusion_coefficient
 
 
-def calculate_self_diffusion_coefficient(MSD):
+def calculate_self_diffusion_coefficient(MSD, t0):
     """
     Take a list of mean square displacement for each timestep.
 
     arguments:
         MSD - list
+        t0 - integer
```

Comment:
While you are here, it would be nice if you explain what the parameters mean, not only their type to improve the documentation.